index.html

```diff
@@ -40,32 +48,91 @@
       }
 
       #output-div {
-        background-color: lightgrey;
+        background-color: rgba(241, 232, 190, 0.521);
         margin: 20px auto;
         padding: 20px;
         width: 100%;
       }
     </style>
   </head>
 
+  <!-- The <body> element defines the document's body, and is a container for all the visible contents that will be rendered by the browser.  -->
   <body>
-    <h1 id="header">Basics: Beat That! 🚀</h1>
+    <!-- The <h1> element defines a large heading. There are 6 pre-set header elements, <h1> to <h6> -->
+    <h1 id="header">🎲🎲 Beat That! 🎲🎲</h1>
+    <!-- The <div> tag defines a division or a section in an HTML document, and is commonly use as a "container" for nested HTML elements. -->
     <div id="container">
-      <p>Input:</p>
+      <!-- The <p> tag defines a paragraph element. -->
+      <p
+        style="
+          font-family: Georgia, 'Times New Roman', Times, serif;
+          font-size: 30px;
+          font-style: bold;
+        "
+      >
+        Welcome to Beat That!
+      </p>
+      <p
+        style="
+          font-family: Georgia, 'Times New Roman', Times, serif;
+          font-size: 20px;
+          font-style: italic;
+        "
+      >
+        To start, please click the "submit" button!
+      </p>
+      <p
+        style="
+          font-family: Georgia, 'Times New Roman', Times, serif;
+          font-size: 16px;
+        "
+      >
+        Input:
+      </p>
+      <!-- The self-closing <input /> tag represents a field for a user to enter input. -->
       <input id="input-field" />
+      <!-- The self-closing <br /> tag represents a line break - a line's worth of spacing before dispalying the next element. -->
       <br />
+      <!-- The <button> tag represents.. you guessed it! a button. -->
       <button id="submit-button">Submit</button>
       <p>Output:</p>
       <div id="output-div"></div>
+      <p>
+        <center>How is this game played?🤔</center>
+      </p>
+      <p>
+        <center>
+          Each player rolls 2 dice and try to arrange them to the highest number
+          possible
+        </center>
+      </p>
+      <p>
+        <center>
+          For example, if you roll a 1 and 5, the highest number possible would
+          be 51
+        </center>
+      </p>
+      <p>
+        <center>
+          The player with the highest arranged number wins, so grab a buddy and
+          let's play! 😝
+        </center>
```

Comment:
Great job playing around with the HTML file and adding instructions.
However, from my knowledge the center tags: <'center'>, is deprecated.
A better way to centralise text is to style it using CSS: https://www.w3schools.com/html/tryit.asp?filename=tryhtml_styles_text-align

What deprecated code is: Deprecated means that it is still in use, but only for historical purposes and it will be removed probably in the next big release. It is recommended that you do not use deprecated functions or features - even if they are present in the current library for example.